plugins/woocommerce/src/Internal/Admin/Notes/WooSubscriptionsNotes.php

```diff
@@ -301,9 +301,10 @@ public function add_or_update_subscription_expiring( $subscription ) {
 
 				foreach ( (array) $bump_thresholds as $bump_threshold ) {
 					if ( ( $note_days_until_expiration > $bump_threshold ) && ( $days_until_expiration <= $bump_threshold ) ) {
```

Comment:
The extremely similarly named existing variables names `$note_days_until_expiration` and `$days_until_expiration` make it extremely difficult for my brain to actually understand what is going on in this code. 🤯